.github/workflows/mutation-testing.yml

```
@@ -5,6 +5,8 @@ on:
     branches:
       - main
 
+permissions: read-all
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)